Sources/SourceKitLSP/Swift/CodeCompletionSession.swift

```diff
@@ -134,7 +158,6 @@ class CodeCompletionSession {
           session.snapshot.uri == snapshot.uri
           && session.position == completionPosition
           && session.compileCommand == compileCommand
-          && session.clientSupportsSnippets == clientSupportsSnippets
```

Comment:
Do we need to check whether the capabilities match, or does that not matter?

Reply:
I don’t think that we need to. The only case where this would change is if the client initiated a code completion session and then dynamically registered the snippet support capability, which would be very unlikely since any of that dynamic registration would happen all up-front. And if should get it wrong, it’s not going to be a fatal error, the results might just look a little off (eg. no snippets even though client supports them).